### ISSUE TITLE
I have been banned for "stealing", following abuse of powers from a moderator

### ISSUE THREAD

**Issue by Howesey** [ObliqueNET/Server#474]
(Terribly sorry if there was a format I needed to use for a ban appeal, but I was not able to find one)

My In-game name is Hwsy and I have recently been banned from the Oblique server and I feel as though I was unfairly reprimanded. The reasons given are that I allegedly "Mass Raided", "Refused to cooperate" and "stated they did not steal anything". After reading through server rules, as listed on the Oblique Discord server, it is evident to me that I did not break any rules.

The ban follows an approximate 4 minute /msg conversation with a Moderator that goes by the name "CorruptedShadow" (as seen on Discord) or "WickedShadow(?)" "(n-game). This piece of information is paricularly relevant, as I am led to beleive that the Moderator in question used the moderator commands or powers available to them for personal gain. This comes after the realisation that the town I was apart of was created and is led by Corrupted Shadow himself.

In the Word documents attached are the chat logs between the Moderator that banned me and myself, as well as the full chat log from all players. It clearly shows timestamped messages to and from Corrupted Shadow. These show the full dialogue between both them and I.

[Chat logs of dialogue between CorruptedShadow and Hwsy.docx](https://github.com/ObliqueNET/Server/files/4451821/Chat.logs.of.dialogue.between.CorruptedShadow.and.Hwsy.docx)
[Full Chat Log of Oblique Server.docx](https://github.com/ObliqueNET/Server/files/4451824/Full.Chat.Log.of.Oblique.Server.docx)

Before we go onto my statements, I'd like it known that i had set two individual homes at two individual bases. The first home was at my plot, where I would keep my PC and Healing Station, and where I had built my house for the town. The second home I had set, was at an undergorund base full of chests. This is where I would store my items. You may be wondering why I have a separate unclaimed base. It is used to store my items so they are not lost in the event of being kicked from the town and unable to retrieve belongings from my town plot. In the attached chat logs, CorruptedShadow states that they can view my of of homes and identify that I have previously stored items at the home listed as "base".

The first point I would like to make is how i was greeted. "You gonna return our shit" (You can view this message in the attached chat logs, timestamped 16:39:33). This first point isn't the most significant, but it is extremely unprofessional to use such blunt and vulgar language as a moderator. As someone that has had many previous moderating experiences, if I were to do that on big vanilla minecraft servers, such as Arkham Network, I would have been dismissed immediately. This reinforces the assumption that this Moderator's actions were for personal gain, as seen in the angry nature of the message.

The second point I would like to make is that the statement that I "refused to cooperate" is false. I was told to "return all items" I had taken from the town and I had never made it clear whether I would have or not, which I admit is a mistake on my part. However, while frozen in place and unable to warp anywhere, I was questioning the actions of the moderator, whether it was for their personal gain or if it was benefiting the server by reprimanding me, as well as questioning how ethical this moderators actions were. So as far as I am concerned, I was co-operating with this moderator and due to this, the ban came as quite a shock to me, a player who recently donated to the server. During this exchange, the moderator messaged "thats what they did, they refused to return the stuff and poof" (this particular message can be seen in the attached chat logs, and is timestamped 16:42:51) and "so i suggest you return the items" (This chat log can be viewed at the timestamp 16:43:03). Personally I viewed this message as a threat, which again shows that the action sof the staff member were for personal gain and not benefiting the server in the slightest.

On to my third point, in the ban message i was given the reason, as stated previously, "Refused to cooperate". I would like to direct your attention to that chat logs attached, timestamped 16:43:58 and 16:44:13 respectively; CorruptedShadow messaged "so is that a yes to returning the items or no", to which I replied with "how does one return items if they cant move or warp". This comes just before I questioned the ethics of this staff member, and no clear answer was given to either one of us. This shows that it is unclear whether or not I was co-operating with this individual and what my intentions were following this interaction. Therefore, the statement that i "refused to cooperate" is made null and void, as I was answering the majority of this Moderator's quesitons whilst also asking them questions in return. As stated previously, I have had many experiences being a Senior Moderator, Moderator and Helper on a handful of servers and the idea is to answer the players' questions fairly and competently.

Finally, my third point, every town has plots and near this Moderator's plot were some unmarked chests with a few held items in them, as well as a few resources gained from mining. As a member of this town, I thought it was acceptable to use these chests for personal use, as well as to aid the development of the town base. Following this, I used some of the items from the chest, as well as took a few held items and whatnot for personal use, after all, the idea of a town is to help support thier residents, correct? If another member of the town had told me to return these items and that they were not to be used by anyone other than a certain player or players, then I would have returned them. So, to simplify this, I was under the impression that all items in these chests could be used freely by any member of the town.

To conclude, when looking over the reasons in which I was banned, I feel as though they are all null and void and my reasons for this mindset are explained in the points above. Furthermore, I would like to express my sadness in being banned from this server, as I was enjoying my time playing on this for the past week which was shown in me donating money to the server. I, also, am not looking for CorruptedShadow to be demoted, as that is not my right to do so. However, I do feel as though a high level of unprofessionalism has been displayed, and the Moderator's morals and ethics must be questioned.

I excitingly await your response, and hope we can get this issue resolved and get back to playing as soon as possible.

**Comment by Rasgnarok** [comment]
I will review your attachments and discuss with the moderator in question once my moderation admin, Elemental, is back online. I'll post developments here.

**Comment by Rasgnarok** [comment]
I've finalized reviewing all of the documents you attached, and most importantly, the logs of the chests and the server itself. The chests you looted included some key ones that were not town-shared, and all items were moved off the location to another base, out of the town's reach. That said, it is evident that this could have been handled better, which is why I will not disagree on your cooperation point. 
Considering this is your first grief instance, I will be moving this ban to a temporary one that will expire on Monday, EST.
